tests/content-addressed.nix

```diff
@@ -60,4 +60,15 @@ rec {
       echo ${rootCA}/non-ca-hello > $out/dep
     '';
   };
+  dependentFixedOutput = mkDerivation {
```

Comment:
Shouldn't there be a build of this attribute in `tests/content-addressed.sh`?

Reply:
Uh of course :man_facepalming: 

Fixed in 9bd8184f1fb2e91ac4fb7207abe56f2a30a81d97